draft-ietf-rtcweb-jsep.xml

```diff
@@ -694,35 +692,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
         <section title="Creating an imageattr Attribute">
 
-          <t>In order to determine the limits on what video resolution
-          a receiver wants to receive, it will intersect its decoder
-          hard limits with any mandatory constraints that have been
-          applied to the associated MediaStreamTrack. If the decoder
-          limits are unknown, e.g. when using a software decoder, the
-          mandatory constraints are used directly. For the answerer,
-          these mandatory constraints can be applied to the remote
-          MediaStreamTracks that are created by a setRemoteDescription
-          call, and will affect the output of the ensuing createAnswer
-          call. Any constraints set after setLocalDescription is used
-          to set the answer will result in a new offer-answer exchange.
-          For the offerer, because it does not know about any remote
-          MediaStreamTracks until it receives the answer, the offer can
-          only reflect decoder hard limits. If the offerer wishes to
-          set mandatory constraints on video resolution, it must do so
-          after receiving the answer, and the result will be a new
-          offer-answer to communicate them.</t>
-
-          <t>If there are no known decoder limits or mandatory
-          constraints, the "a=imageattr" attribute SHOULD be
-          omitted.</t>
+          <t>The receiver will first intersect any known local limits (e.g.,
+          hardware decoder capababilities, local policy) to determine the
+          absolute minimum and maximum sizes it can receive.
+          If there are no known local limits, the "a=imageattr" attribute
+          SHOULD be omitted.</t>
 
           <t>Otherwise, an "a=imageattr" attribute is created with
           "recv" direction, and the resulting resolution space formed
-          by intersecting the decoder limits and constraints is used to
+          from the aforementioned intersection is used to
           specify its minimum and maximum x= and y= values. If the
-          intersection is the null set, i.e., there are no resolutions
-          that are permitted by both the decoder and the mandatory
-          constraints, this MUST be represented by x=0 and y=0
+          intersection is the null set, i.e., the degenerate case of no
+          permitted resolutions, this MUST be represented by x=0 and y=0
```

Comment:
The new text has local policy + hardware limits as the only things to intersect. It would be weird to have those end up with the empty set, but the language seems to me to be be OK for handling this weird case.